lib/OpenQA/BuildResults.pm

```diff
@@ -0,0 +1,117 @@
+# Copyright (C) 2016 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+package OpenQA::BuildResults;
+use strict;
+use OpenQA::Schema::Result::Jobs;
+use Date::Format;
+
+sub compute_build_results {
+    my ($app, $group, $limit, $time_limit_days) = @_;
```

Comment:
This is not necessary anyway as openQA::Utils provides log_debug for such cases.